developer_setup.md

```diff
@@ -199,6 +203,13 @@
 * Required Ruby version is 2.2.2+
 * Required Bundler version is 1.8.7+
 
+* Steps to install Ruby & Bundler using `rvm`
+
+  ```bash
+\curl -sSL https://get.rvm.io | bash -s stable --rails
+rvm install \<Ruby_version\>
+  ```
+
```

Comment:
This is why I didn't want the rbenv line below because it's just a neverending list of ways to install Ruby :/  (not your fault @d-m-u  😄 )

Reply:
Yeah, I get you, but I dunno what to do about it anyhow. Chruby! I missed it! Well, gosh, darn.